.github/workflows/react_demos.yml

```
@@ -0,0 +1,56 @@
+name: React Demos
+
+on:
+  push:
```

This lets us manually trigger the actions from GitHub and is a huge help when testing new actions
```suggestion
  workflow_dispatch:
  push:
```

---

Good idea, i would create a dummy pull request for such a thing!